classes/TaskRunner/Miscellaneous/UpdateConfig.php

```diff
@@ -95,6 +108,40 @@ public function run()
         }
     }
 
+    /**
+     * @param array $parameters
+     */
+    public function inputCLIParameters($parameters)
+    {
+        $this->isRunInCLI = true;
+        $this->cliParameters = $parameters;
+    }
+
+    /**
+     * Fetch data from either $_REQUEST (web context) or given input (CLI context)
+     */
+    protected function fetchConfigurationData()
```

Comment:
```suggestion
    protected function fetchConfigurationData(): array
```

Reply:
Can't use PHP 7 features in the autoupgrade module